### PR TITLE
`Exam mode`: Fix missing spacing at the end of exercise in exams 

### PR DESCRIPTION
**Pull request by mateusmm01**
#### General

- [x] I tested **all** changes and their related features with **all** corresponding user types on a test server.
- [x] This is a small issue that I tested locally and was confirmed by another developer on a test server.
- [x] Language: I followed the [guidelines for inclusive, diversity-sensitive, and appreciative language](https://docs.artemis.cit.tum.de/dev/guidelines/language-guidelines/).
- [x] I chose a title conforming to the [naming conventions for pull requests](https://docs.artemis.cit.tum.de/dev/development-process/#naming-conventions-for-github-pull-requests).

#### Changes affecting Programming Exercises
- [ ] **High priority**: I tested **all** changes and their related features with **all** corresponding user types on a test server configured with the **integrated lifecycle setup** (LocalVC and LocalCI).
- [ ] I tested **all** changes and their related features with **all** corresponding user types on a test server configured with **Gitlab** and **Jenkins**.

### Motivation and Context

When creating programming exercise **without** online editor the problem statement gets truncated at the end of the page. This shouldn't be like that
image

#### Exam Mode Testing

Prerequisites:
- 1 Instructor
- 1 Students
- 1 Exam with a Programming Exercise **without** Online Editor

1. Log in to Artemis
2. Participate in the exam as a student
3. Make sure that the UI of the programming exercise in the exam mode now shows the whole problem statement when opening the exrércise.

#### Performance Review
- [ ] I (as a reviewer) confirm that the client changes (in particular related to REST calls and UI responsiveness) are implemented with a very good performance 
- [ ] I (as a reviewer) confirm that the server changes (in particular related to database calls) are implemented with a very good performance
#### Code Review
- [ ] Code Review 1
- [ ] Code Review 2
#### Exam Mode Test
- [ ] Test 1
- [ ] Test 2

### CODE REVIEW

**Review by laurenzfb — DISMISSED** [review]
code lgtm

**Review by rstief — DISMISSED** [review]
code looks ok 👀

File: src/main/webapp/app/exam/participate/exam-participation.component.html [hunk 1]
```diff
@@ -28,7 +28,7 @@
             @for (exercise of studentExam.exercises; track exercise; let i = $index) {
                 @if (exercise && exercise.studentParticipations && exercise.studentParticipations[0]) {
                     @if (pageComponentVisited[i]) {
-                        <div [hidden]="i !== activePageIndex" [id]="'exercise-' + exercise.id">
+                        <div style="margin-bottom: 30px" [hidden]="i !== activePageIndex" [id]="'exercise-' + exercise.id">
```
Comment (Strohgelaender) [inline src/main/webapp/app/exam/participate/exam-participation.component.html:31]: We prefer classes over inline-css for these purposes (not sure if 2 is the correct value to choose from, but is should be some `mb-x` class)
```suggestion
                        <div class="mb-2" [hidden]="i !== activePageIndex" [id]="'exercise-' + exercise.id">
```

Comment (rabeatwork) [reply src/main/webapp/app/exam/participate/exam-participation.component.html:31]: `mb-4` would be 1.5rem. This equals at the moment 24px.

**Review by rstief — APPROVED** [review]

**Review by rabeatwork — APPROVED** [review]
maintainer approve

**Pull request merged**